src/pages/projectSettings/tabs/EventLogTab/useEvents.ts

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+import { useQuery } from "@apollo/client";
+import { useToastContext } from "context/toast";
+import {
+  ProjectEventLogsQuery,
+  ProjectEventLogsQueryVariables,
+  RepoEventLogsQuery,
+  RepoEventLogsQueryVariables,
+} from "gql/generated/types";
+import { GET_PROJECT_EVENT_LOGS, GET_REPO_EVENT_LOGS } from "gql/queries";
+
+export const useEvents = (
+  identifier: string,
+  isRepo: boolean,
+  limit: number
+) => {
+  const dispatchToast = useToastContext();
+
+  const [prevCount, setPrevCount] = useState(0);
+  const [showLoadButton, setShowLoadButton] = useState(true);
+
+  // Hide Load More button when event count < limit is returned,
+  // or when the same event count is returned twice in a row.
+  const onCompleted = (eventData) => {
+    if (eventData.length === prevCount) {
+      setShowLoadButton(false);
+    } else if (eventData.length - prevCount < limit) {
+      setShowLoadButton(false);
+    } else {
```

Comment:
this may be a silly, silly question... but I am wondering what scenario requires the first if-statement... because I feel like it could be written equivalently as `eventData.length - prevCount === 0` which seems like it would fall under `eventData.length - prevCount < limit` as long as `limit` is positive 🤔